examples/2d/mesh2d_circular.rs

```diff
@@ -0,0 +1,128 @@
+
+        let sector = CircularSector::from_fraction(40.0, fraction);
+        // We want to rotate the circular sector so that the sectors appear clockwise from north.
+        // We must rotate it both in the Transform and in the mesh's UV mappings.
+        let sector_angle = -sector.half_angle();
+        let sector_mesh =
+            CircularSectorMeshBuilder::new(sector).uv_mode(CircularMeshUvMode::Mask {
+                angle: sector_angle,
+            });
+        commands.spawn((
+            MaterialMesh2dBundle {
+                mesh: meshes.add(sector_mesh).into(),
+                material: material.clone(),
+                transform: Transform {
+                    translation: Vec3::new(FIRST_X + OFFSET * i as f32, 2.0 * UPPER_Y, 0.0),
+                    rotation: Quat::from_rotation_z(sector_angle),
+                    ..default()
+                },
+                ..default()
+            },
+            DrawBounds(sector),
+        ));
+
+        let segment = CircularSegment::from_fraction(40.0, fraction);
+        // For the circular segment, we will draw Bevy charging forward, which requires rotating the
+        // shape and texture by 90 degrees.
+        //
+        // Note that this may be unintuitive; it may feel like we should rotate the texture by the
+        // opposite angle to preserve the orientation of Bevy. But the angle is not the angle of the
+        // texture itself, rather it is the angle at which the vertices are mapped onto the texture.
+        // so it is the negative of what you might otherwise expect.
+        let segment_angle = -PI / 2.0;
+        let segment_mesh =
+            CircularSegmentMeshBuilder::new(segment).uv_mode(CircularMeshUvMode::Mask {
+                angle: -segment_angle,
+            });
+        commands.spawn((
+            MaterialMesh2dBundle {
+                mesh: meshes.add(segment_mesh).into(),
+                material: material.clone(),
+                transform: Transform {
+                    translation: Vec3::new(FIRST_X + OFFSET * i as f32, LOWER_Y, 0.0),
+                    rotation: Quat::from_rotation_z(segment_angle),
+                    ..default()
+                },
+                ..default()
+            },
+            DrawBounds(segment),
+        ));
+    }
+}
+
+fn draw_bounds<Shape: Bounded2d + Send + Sync + 'static>(
+    q: Query<(&DrawBounds<Shape>, &GlobalTransform)>,
+    mut gizmos: Gizmos,
+) {
+    for (shape, transform) in &q {
+        let (_, rotation, translation) = transform.to_scale_rotation_translation();
+        let translation = translation.truncate();
+        let rotation = rotation.to_euler(EulerRot::XYZ).2;
+
+        let aabb = shape.0.aabb_2d(translation, rotation);
+        gizmos.rect_2d(aabb.center(), 0.0, aabb.half_size() * 2.0, Color::RED);
+
+        let bounding_circle = shape.0.bounding_circle(translation, rotation);
+        dbg!(bounding_circle);
```

Comment:
Is this intended to stay here? I'm guessing it was just for debugging and can be removed
```suggestion
```